src/travis2docker/travis2docker.py

```diff
@@ -112,27 +112,17 @@ def _compute(self, section):
         if not section_type:
             return None
         section_data = self.yml.get(section, "")
-        if section != "env" and not section_data:
+        if section != 'env_global' and not section_data:
             return None
         if not isinstance(section_data, (list, dict, tuple)):
             section_data = [section_data]
         job_method = getattr(self, '_compute_' + section_type)
         return job_method(section_data, section)
 
-    @staticmethod
-    def _compute_env(data, _):
-        if isinstance(data, list):
-            # old version without matrix
-            data = {'matrix': data}
-        env_globals = ""
-        for env_global in data.get('global', []):
-            if isinstance(env_global, dict):
-                # we can't use the secure encrypted variables
-                continue
-            env_globals += " " + env_global
-        env_globals = env_globals.strip()
-        for env_matrix in data.get('matrix', []):
-            yield (env_globals + " " + env_matrix).strip()
+    def _compute_env_global(self, data, _):
+        globals = (self.yml.get('env').get('global')
```

Comment:
Isn't a built-in variable `globals`?

Reply:
@moylop260 
Good catch, I change the name of the variable to avoid replace the build-in function globals